[2022-12-12 18:39:28,483] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:19:00+00:00 [queued]>
[2022-12-12 18:39:28,644] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:19:00+00:00 [queued]>
[2022-12-12 18:39:28,646] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 18:39:28,656] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 18:39:28,657] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 18:39:28,808] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_last_id> on 2022-11-28 17:19:00+00:00
[2022-12-12 18:39:28,825] {standard_task_runner.py:52} INFO - Started process 457 to run task
[2022-12-12 18:39:28,930] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'get_last_id', 'scheduled__2022-11-28T17:19:00+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpbyni2j8i', '--error-file', '/tmp/tmp__wkyrsc']
[2022-12-12 18:39:28,943] {standard_task_runner.py:80} INFO - Job 122: Subtask get_last_id
[2022-12-12 18:39:29,729] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.get_last_id scheduled__2022-11-28T17:19:00+00:00 [running]> on host cfea470654c7
[2022-12-12 18:39:31,283] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=get_last_id
AIRFLOW_CTX_EXECUTION_DATE=2022-11-28T17:19:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-28T17:19:00+00:00
[2022-12-12 18:39:31,506] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-12 18:39:31,779] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 18:39:32,061] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=get_last_id, execution_date=20221128T171900, start_date=20221212T183928, end_date=20221212T183932
[2022-12-12 18:39:32,192] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 18:39:32,397] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
